Source/WebCore/css/ComputedStyleExtractor.cpp

```diff
@@ -1268,6 +1268,9 @@ static Ref<CSSValue> valueForScrollbarGutter(const ScrollbarGutter& gutter)
 
 static Ref<CSSValueList> valueForTextBoxEdge(const TextBoxEdge& textBoxEdge)
 {
+    if (textBoxEdge.over == TextBoxEdgeType::Leading && textBoxEdge.under == TextBoxEdgeType::Leading)
```

Comment:
@nt1m Is there a reason this condition shouldn't simply be `textBoxEdge.over == textBoxEdge.under`?

There is another failing test which says:
```
FAIL Property text-box-edge value 'text' assert_equals: expected "text" but got "text text"
```